tests/rptest/tests/mirror_maker_test.py

```diff
@@ -82,6 +82,8 @@ def start_brokers(self, source_type=kafka_source):
         self.redpanda.start()
 
         self.source_client = DefaultClient(self.source_broker)
+
+        self.topic.partition_count = 1000 if self.redpanda.dedicated_nodes else 1
```

Comment:
:+1: Nice to see some chunky partition counts.  At some point in the near future (probably once we have nightly execution for clustered ducktape) we might want to move this test into scale_tests/